CHANGELOG.next.asciidoc

```diff
@@ -119,6 +119,7 @@ is collected by it.
 
 - Fix panics when parsing dereferencing invalid parsed url. {pull}34702[34702]
 - Fix retries to trigger on a down monitor with no previous state. {pull}36842[36842]
+- Bump NodeJS minor version to 18.18.x. {pull}36961[36961]
```

Comment:
Because the node version is pinned to a specific one, I believe it's better to have the full version here:
```suggestion
- Bump NodeJS minor version to 18.18.2. {pull}36961[36961]
```

Reply:
@belimawr, pushed your suggestion. Please review and unblock if OK